pkgs/servers/monitoring/icinga2/default.nix

```
@@ -0,0 +1,94 @@
+{ stdenv, lib, fetchFromGitHub, cmake, flex, bison, systemd, makeWrapper
+, boost, openssl, monitoring-plugins, mariadb-connector-c, postgresql, zlib
```

```suggestion
, boost, openssl, mariadb-connector-c, postgresql, zlib
```
That argument is unused.